### PR TITLE
WIP: PAT-473: Change Helm chart repository from cammis to hello-cammis

### PR DESCRIPTION
**Pull request by krenshaw2018**
This trivial change is to change the Helm chart repository that is pulled from from cammis -> hello-cammis.

### CODE REVIEW

File: helm/hello-cammis-api/requirements.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 dependencies:
   - name: hello-cammis-data
     version: 1.0.1
-    repository: "@cammis"
+    repository: "@hello-cammis"
```
[the remark this answers is not in the document]
Comment (krenshaw2018) [reply helm/hello-cammis-api/requirements.yaml]: Reverted.

**Comment by krenshaw2018** [comment]
Hi Aaron, this last re-review request is to get the MASTER_PLAN fix in place. Thanks.

File: .circleci/config.yml [hunk 2]
```diff
@@ -140,7 +140,7 @@ jobs:
             # Local env vars are not shared between "run" stanzas
             if [ "${CIRCLE_BRANCH}" ] ; then
               REVIEW_ENV_NAME="review_${CIRCLE_PROJECT_REPONAME}_br_${CIRCLE_BRANCH}"
-              #MASTER_CHART_NAME=cammis  # default is cammis
+              MASTER_CHART_NAME=hello-cammis  # default is cammis
```
Comment (gubbin) [inline .circleci/config.yml]: Looks good.

**Comment by krenshaw2018** [comment]
Per @gubbin I ran review env builds for all 3 hello-cammis-* repos, branched into r_PAT-473-krenshaw branches. In all cases the review environments spun up correctly and the resultant webpage on the UI rendered fine. Closing and merging.

**Pull request merged**